src/dmd/expression.d

```diff
@@ -1072,7 +1072,13 @@ extern (C++) int expandAliasThisTuples(Expressions* exps, size_t starti = 0)
 }
 
 /****************************************
- * Get TemplateDeclaration enclosing FuncDeclaration.
+ * If `s` is a function template, i.e. the only member of a template
+ * and that member is a function,
+ * return that template.
```

Comment:
Why is there a newline after the comma? Looks like `return that template` would fit on the previous line.